resources/assets/sass/plugins/_password-strength.scss

```diff
@@ -0,0 +1,44 @@
+.strengthify-wrapper > * {
+    -ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: alpha(opacity=0);
+    opacity: 0;
+    -webkit-transition:all .5s ease-in-out;
+    -moz-transition:all .5s ease-in-out;
+    transition:all .5s ease-in-out;
+}
+
+.strengthify-wrapper {
+    position: relative;
+}
+
+.strengthify-bg, .strengthify-container, .strengthify-wrapper, .strengthify-separator {
+    height: 5px;
+}
+
+.strengthify-bg, .strengthify-container {
+    display: block;
+    position: absolute;
+    width: 100%;
+}
+
+.strengthify-bg {
+    background-color: #BBB;
+}
+
+.strengthify-separator {
+    display: inline-block;
+    position: absolute;
+    background-color: #FFF;
+    width: 1px;
+    z-index: 10;
+}
+
+.password-bad {
+    background-color: #C33;
```

Comment:
@AntoineAugusti Can you use our palette colours? https://github.com/cachethq/Cachet/blob/master/resources/assets/sass/_palette.scss